handbook/marketing/content.md

```diff
@@ -4,13 +4,51 @@ At Sourcegraph, we define **content marketing** as generating awareness, buildin
 
 **Content strategy** dives a layer deeper to include the governance and operations of content: how we generate, validate, and approve ideas; how we ensure quality and consistency; where we distribute content; how we measure performance; and how we update, revise, and retire content.
 
+[Read the FY22 Content Strategy.](https://docs.google.com/document/d/1uz9AN5ZiGkbVD5mQmoKNO4mdbKaI1lukQcxPB2JiPx4/edit#)
+
+## Table of contents 
+
+1. [Quick links]
+1. [Editorial process]
+
 ## Quick links
 
 - [Editing blog posts](editing_blog_posts.md)
 - [Creating blog posts](creating_blog_posts.md)
 - [Adding screenshots and screen recordings (GIFs and videos)](adding_screenshots_screen_recording.md)
 - [Website style guide](website_style_guide.md)
 
+## Purpose, mission, and vision
+
+Our purpose, mission, and vision explain why we create, who we create it for, and what we strive to achieve. 
+
+| ----------- | ----------- |
```

Comment:
```suggestion
| ----------- | ----------- |
| ----------- | ----------- |
```
Since this table has no headers, this is the workaround, doubling the lines with separators.

Reply:
aaah, good to know! thank you!